src/app/stores/store.ts

```diff
@@ -172,6 +172,38 @@ export class Store {
       .find(response => response.uuid === this.store$.value.activeRouteResponseUUID);
   }
 
+  /**
+   * Generate a unused port to a new environment
+   */
+  public getNewEnvironmentPort(): number {
+    const usedPorts: number[] = [];
+    this.store$.value.environments.forEach((environment) => {
+      usedPorts.push(environment.port);
+    });
+
+    const activeEnvironment: Environment = this.getActiveEnvironment();
+    let testSelectedPort: number;
+    if (activeEnvironment == null) {
+      testSelectedPort = 3000;
+    } else {
+      testSelectedPort = activeEnvironment.port;
+    }
+    for (let i = 0; i < 10; i++) {
```

Comment:
This make sense if we want to continue the serie 3000, 3001, 3002. But if one chose the port 65535, it will increase to 65536 and so on. I know there is no limit on the input field but we should add a check to avoid this. Or directly use the random method below.